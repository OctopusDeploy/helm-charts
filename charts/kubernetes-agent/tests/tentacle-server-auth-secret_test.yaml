suite: "tentacle configuration"
templates:
  - templates/tentacle-server-auth-secret.yaml
tests:
  - it: "should match snapshot"
    asserts:
      - matchSnapshot: {}