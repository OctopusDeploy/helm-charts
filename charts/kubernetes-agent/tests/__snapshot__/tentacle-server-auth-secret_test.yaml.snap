should match snapshot:
  1: |
    apiVersion: v1
    data: null
    kind: Secret
    metadata:
      name: octopus-agent-tentacle-server-auth
      namespace: NAMESPACE
    type: Opaque
